lib/test-runner/qunit/acceptance.js

```diff
@@ -1,14 +1,12 @@
 function testFeature(feature) {
 
-  // if a runFeature function is provided, use it to determine if should run the feature
-  if (typeof(runFeature) === 'function' && !runFeature(feature.annotations)) {
-    console.log('not running feature: ', feature.title);
+  let runFeature = yaddaAnnotations.runFeature(feature.annotations);
```

Comment:
if `runFeature` would return a function and if we'd call that function from the current scope would we be able to access all the stuff in the current scope?

i.e.:

```
if (annotations.ignore) {
  return function() {
    skip(`Feature ${feature.title}`, function (assert) {});
  }
}
```

This way you could do anything to the test, not just skip it. I'm not sure if we could teach this to the user though. We definitely should rely on them having to do this stuff: 

          test('Scenario: ${scenario.title}', function(assert) {
            let self = this;
            return new Ember.RSVP.Promise(function(resolve) {
              yadda.Yadda(library.default(assert), self).yadda(scenario.steps, { ctx: {} }, function next(err, result) {
                if (err) {
                  throw err;
                }
                resolve(result);
              });
            });

I don't know how that would work exactly. We will definitely have to change the mocha runner to do this annotations dance too though.